share/revng/test/configuration/revng-qa/for-detect-abi.yml

```diff
@@ -77,6 +82,11 @@ sources:
       - [i386-regparm-3-systemv-abi]
       - [i386-regparm-2-systemv-abi]
       - [i386-regparm-1-systemv-abi]
+      - [x86-64-systemv-abi, intel-asm-syntax]
+      - [i386-systemv-abi, intel-asm-syntax]
+      - [i386-regparm-3-systemv-abi, intel-asm-syntax]
+      - [i386-regparm-2-systemv-abi, intel-asm-syntax]
+      - [i386-regparm-1-systemv-abi, intel-asm-syntax]
```

Comment:
1. More like to make the compiler use exclusively `intel syntax`, I don't remember the specifics, but I have read somewhere that it's a bad idea to mix them - there are some minor incompatibilities.

2. First of all, we don't use inline assembly anymore. But also, I kind of prefer how explicit flag feels.

One option would be to try and move this flag higher up or something instead of introducing a separate tag for it or something.

P. S. I don't have any strong opinions on this. As long as the compiler is happy - so am I.

Reply:
This change is not present in the final diff. Why is that?
Please squash fixup'd commits.

Reply:
Looks like I lost it somewhere along the way, now I'm really confused about why it even works. There's probably a rogue flag hardcoded somewhere or something. I'll investigate tomorrow.

Reply:
Done.